spec/features/super_admin/whitelists/organisation_names/add_custom_organisation_spec.rb

```diff
@@ -1,10 +1,27 @@
-describe 'Adding a custom organisation name', type: :feature do
+describe 'Adding a custom organisation name', type: :feature, focus: true do
   let(:admin_user) { create(:user, :super_admin) }
 
   before do
     sign_in_user admin_user
   end
 
+  context 'when adding a custom organisation'do
+    let(:name) { 'Custom Org name' }
```

Comment:
Think this test will always pass because of the hardcoded name in your presenter/usecase object

Reply:
Yeah, just wanted it passing for now! Baby steps!